### PR TITLE
Update jedi-guardian.md

### PR DESCRIPTION
**Pull request by jetfire158**

### CODE REVIEW

File: content/theme/characters/archetypes/jedi/jedi-guardian.md [hunk 1]
```diff
@@ -2,5 +2,41 @@
...
+     dodge: 3
+     leap: 3
+     melee: 4
+ ---
+
+Leaders both on and off the battlefield, Jedi Guardians stand valiant and determined as protectors of the galaxy, sometimes against overwhelming odds. Some of the Order's best warriors are Guardians who have honed their minds and bodies into perfect harmony through years of Force attunement, training, and meditation. 
+
+Inspiring allies to incredible feats and striking fear into the hearts of their foes, Guardians are critical to the preservation and continued well-being of the Galactic Republic.
+
+--
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/jedi/jedi-guardian.md]: Need to remove this `--`

**Review by jetfire158 — COMMENTED** [review]
fixed

File: content/theme/characters/archetypes/jedi/jedi-guardian.md [hunk 2]
```diff
@@ -2,5 +2,38 @@
...
+   presence:
+     willpower: 3
+     intimidate: 2
+     lead: 3
+   reflex:
+     acrobatics: 4
+     dodge: 3
+     leap: 3
+     melee: 4
+ ---
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/jedi/jedi-guardian.md]: Sorry, also should not be a leading space before the front-matter delimiter `---`
Had to do some searching to find out why the build was failing.

**Review by jetfire158 — COMMENTED** [review]
ack, those evil errant spaces!

**Pull request closed without merging**